### ISSUE TITLE
[CYS] Intro screen - Color swatches

### ISSUE THREAD

**Issue by chihsuan** [woothemes/woocommerce#41020]
Veronica: Color swatches don't work—I know it's a Dotcom issue and there's no need to prioritize it for the MVP launch. Just adding it here for future iterations.

<img width="1320" alt="Intro screen" src="https://github.com/woocommerce/woocommerce/assets/4344253/36afb0d3-80b2-414d-8ef3-d379539e16d3">

**Comment by Aljullu** [comment]
> Color swatches don't work

@chihsuan do you have more details about what's the expectation here? Should these color swatches be clickable or something? My understanding is that they were only used to showcase the available color palettes.

**Comment by chihsuan** [comment]
@Aljullu Yes, they should be clickable and show the corresponding preview when clicked. You can also refer to Dotcom for the behavior here: https://wordpress.com/setup/free/designSetup?siteSlug=test.wordpress.com

This issue was reported by Veronica. Since it has been a while, I think it might be a good idea to check with her to see if she still needs this feature. 

https://github.com/user-attachments/assets/f0ca9dc1-c73f-4088-9c15-dd1a2cb969af

**Comment by Aljullu** [comment]
Oh, TIL. Thanks @chihsuan! Sure, I will check with Veronica before prioritizing this, just wanted to make sure the issue was still reproducible (which still is).

**Comment by tyxla** [comment]
Closing this as it's no longer relevant since the CYS flow was removed: [https://github.com/woocommerce/woocommerce/issues/61932](https://github.com/woocommerce/woocommerce/issues/61932)
